woodwork/table_accessor.py

```diff
@@ -84,70 +120,72 @@ def init(self,
+        def normalize_standard_tags(use_standard_tags: Union[bool, Dict[str, bool]], column_names: List[str]):
             if isinstance(use_standard_tags, bool):
                 use_standard_tags = {col_name: use_standard_tags for col_name in column_names}
             else:
                 use_standard_tags = {**{col_name: True for col_name in column_names}, **use_standard_tags}
+            return use_standard_tags
 
-            self._schema = TableSchema(column_names=column_names,
-                                       logical_types=parsed_logical_types,
-                                       index=index,
-                                       time_index=time_index,
-                                       validate=validate,
-                                       use_standard_tags=use_standard_tags,
-                                       **kwargs)
+        if validate:
+            _validate_accessor_params(self._dataframe, index, time_index, logical_types, schema, use_standard_tags)
 
-            self._set_underlying_index()
-            if self._schema.time_index is not None:
-                self._sort_columns(already_sorted)
+        existing_logical_types = {}
+        existing_col_descriptions = {}
+        existing_col_metadata = {}
+        existing_use_standard_tags = {}
+        existing_semantic_tags = {}
+        existing_col_origins = {}
+
+        if schema:  # pull schema parameters
+            name = name or schema.name
+            index = index or schema.index
+            time_index = time_index or schema.time_index
+            table_metadata = table_metadata or schema.metadata
+            for col_name, col_schema in schema.columns.items():
+                existing_logical_types[col_name] = col_schema.logical_type
+                existing_semantic_tags[col_name] = col_schema.semantic_tags - {'time_index'} - {'index'}
+                existing_col_descriptions[col_name] = col_schema.description
+                existing_col_origins[col_name] = col_schema.origin
+                existing_col_metadata[col_name] = col_schema.metadata
+                existing_use_standard_tags[col_name] = col_schema.use_standard_tags
+
+        # overwrite schema parameters with specified kwargs
+        column_names = list(self._dataframe.columns)
+        logical_types = _infer_missing_logical_types(self._dataframe, logical_types, existing_logical_types)
+        column_descriptions = {**existing_col_descriptions, **(column_descriptions or {})}
+        column_metadata = {**existing_col_metadata, **(column_metadata or {})}
+
+        # prioritize columns in schema if use_standard_tags is True
+        normalized_tags = normalize_standard_tags(use_standard_tags, column_names)
+        if use_standard_tags is True:
+            use_standard_tags = {**normalized_tags, **existing_use_standard_tags}
+        else:
+            use_standard_tags = {**existing_use_standard_tags, **normalized_tags}
```

Comment:
Not totally sure I'm following what's happening here... But if I understand what the behavior here is:

`use_standard_tags` will be `True` by default, so if that's the case, we assume that that's just the default (and not that the user wanted all columns to use `True`), so we overwrite those values with whatever was in the Schema. This will contradict what the docstring says about the `schema` arg and the `use_standard_tags` arg if the Schema being passed in has some columns as `False`.

What if someone does want to set all of them to True? Not sure that it'd be expected to suddenly have the schema overwrite. I think we might need the default to be None here. That way a user can specify `True` and have it mean `True`.

Can we add some tests that cover this behavior?

Reply:
I think I meant to set the default value for unspecified use_standard_tags columns. I removed this.